stripe/src/main/java/com/stripe/android/paymentsheet/BaseAddCardFragment.kt

```diff
@@ -105,10 +106,16 @@ internal abstract class BaseAddCardFragment(
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        if (activity == null) {
+        val nullableConfig = arguments?.getParcelable<FragmentConfig>(BasePaymentSheetActivity.EXTRA_FRAGMENT_CONFIG)
+        if (activity == null || nullableConfig == null) {
+            sheetViewModel.onFatal(
+                IllegalArgumentException("Failed to start add payment option fragment.")
+            )
             return
         }
 
+        val config = nullableConfig
+
```

Comment:
[optional] This val seems unnecessary. Up to you if you want to keep it though.

Reply:
Will leave for now!